src/bonsai/hooks/useFundingDataProcessing.ts

```
@@ -0,0 +1,213 @@
+    filterPaymentsByDateRange,
+    filterPaymentsByMarket,
+    findLargestPayments,
+    FundingPaymentProcessed,
+    FundingStats,
+    getFundingRateStats,
+    getTimePeriodRange,
+    MarketFundingAggregate,
+    processFundingPayment,
+    TimePeriod,
+} from '../calculators/fundingDataProcessing';
+import { BonsaiCore } from '../ontology';
```

ESLint

[error] Dependency cycle detected.

(import/no-cycle)